libs/vkd3d/resource.c

```diff
@@ -3502,6 +3502,22 @@ static void vkd3d_create_buffer_srv(struct d3d12_desc *descriptor,
         return;
     }
 
+    if (desc->ViewDimension == D3D12_SRV_DIMENSION_RAYTRACING_ACCELERATION_STRUCTURE)
+    {
+        /* We implement this as a raw VA in the aux buffer. */
+        if (device->bindless_state.flags & VKD3D_RAW_VA_AUX_BUFFER)
+        {
+            VkDeviceAddress *counter_addresses = descriptor->heap->raw_va_aux_buffer.host_ptr;
+            uint32_t descriptor_index = d3d12_desc_heap_offset(descriptor);
+            counter_addresses[descriptor_index] = desc->RaytracingAccelerationStructure.Location;
+            descriptor->metadata.flags = VKD3D_DESCRIPTOR_FLAG_RAW_VA_AUX_BUFFER;
```

Comment:
We should reset `cookie` and `set_info_mask` as well or otherwise descriptor copies will end up being weird.

Reply:
Fixed.